src/System.Net.Http.WinHttpHandler/tests/System.Net.Http.WinHttpHandler.Tests.csproj

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@mmitche in a general sense I agree with you but I also believe that there are exceptions to every rule and some cases warrant a need to access the internals via InternalsVisibleTo or through direct importing of source code like is happening here. It certainly limits the re-usability of the tests for multiple implementations of a given API contract but that alone wouldn't make me block such a thing as much as I hate it. I do still believe it should be limited and be the exception and not the rule but for cases like networking where it is extremely difficult to mimic the different states I think it is acceptable.